README.md

```diff
@@ -69,14 +69,28 @@ Here is a quick **[demo](https://terminalizer.com/view/3f24561a4470)** shows how
 
 ## Installation
 
-Users can easily install ``Qlib`` by pip according to the following command(Currently, Qlib only support Python 3.6, 3.7 and 3.8). 
+This table demonstrates the supported Python version of `Qlib`:
+|               | install with pip           | install from source  | plot |
```

Comment:
Make more explanation about `plot`
Only the report plotting  in Analyser in jupyter notebook is not supported